src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/get_system_info_response_test.cc

```diff
@@ -98,28 +121,35 @@ TEST_F(GetSystemInfoResponseTest, GetSystemInfo_SUCCESS) {
 
   ResponseFromHMIPtr command(CreateCommand<GetSystemInfoResponse>(command_msg));
 
-  EXPECT_CALL(mock_policy_handler_,
```

Comment:
@ychernysheva Why do you removed OnGetSystemInfo calls from tests?

Reply:
Because in fact this call doesn't affect result of test executions, and all this calls will be the same in all happy-path tests.